workflow/cron/operator.go

```diff
@@ -251,6 +274,15 @@ func (woc *cronWfOperationCtx) removeFromActiveList(uid types.UID) {
 	woc.cronWf.Status.Active = newActive
 }
 
+func (woc *cronWfOperationCtx) containsActiveWorkflow(uid types.UID) bool {
+	for _, ref := range woc.cronWf.Status.Active {
```

Comment:
minor - you would move this to `woc.cronWf.Status.Active.HasUID(uui)`

Reply:
Sure